packages/eslint-plugin-fbtee/src/utils.tsx

```diff
@@ -107,3 +107,47 @@ export function resolveNodeValue(
 
   return null;
 }
+
+export function propName(prop: TSESTree.JSXAttribute) {
+  if (prop.name.type === 'JSXNamespacedName') {
+    return `${prop.name.namespace.name}:${prop.name.name.name}`;
+  }
+
+  return prop.name.name;
+}
+
+export function hasFbtParent(node: TSESTree.Node) {
+  let current = node.parent;
+
+  while (current) {
+    if (isNodeFbt(current)) {
+      return true;
+    }
+    current = current.parent;
+  }
+
+  return false;
+}
+
+export function isNodeFbt(node?: TSESTree.Node) {
+  if (node?.type === 'JSXElement') {
```

Comment:
I'd recommend short circuiting `if (node == null)` so you don't have to use `name?.` everywhere.